frontend/src/core/comments/components/AddComment.css

```diff
@@ -17,7 +17,6 @@
     margin: auto;
     overflow: auto;
     padding: 7px;
-    word-break: break-all;
```

Comment:
`word-break: break-word;` would be consistent with Comment.css and prevent long words from breaking the editor. For example, try pasting:
```
mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm
```